front-end/assets/css/main.scss

```diff
@@ -0,0 +1,19 @@
+// assets/styles.scss
+$primary-color: #3498db;
+
+body {
```

Comment:
?? 

Add CSS resets instead of adding arbitrary and experimental CSS code. @Sagar-Kap will share an example of CSS resets with you. Remember to add the CSS resets as a SASS partial (named as `_resets.scss`).